mrpt_graphslam_2d/remoteApi/license.txt

```diff
@@ -0,0 +1,34 @@
+All the files in each directory are licensed under the terms of the
```

Comment:
Rename file to LICENSE so that github can detect it ;)

Reply:
Done